### PR TITLE
AJ-999: incorporate Release Drafter action

### PR DESCRIPTION
**Pull request by davidangb**
part of AJ-999 but does not complete that ticket.

Incorporate the [Release Drafter](https://github.com/marketplace/actions/release-drafter) github action, using recommended config, to help us automatically generate drafts for release notes.

I'm suggesting we let this run for a bit and see what it does for us, then we can make a decision on whether or not it'll benefit us. We'll see, after we gather some data!

### CODE REVIEW

File: .github/release-drafter-config.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - 'feature'
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
+      - 'bug'
+  - title: '🧰 Maintenance'
+    label: 'chore'
+change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
```
Comment (yuliadub) [inline .github/release-drafter-config.yml:17]: where do author and number come from?

**Review by yuliadub — APPROVED** [review]

File: .github/release-drafter-config.yml (hunk 1 above)
Comment (yuliadub) [reply .github/release-drafter-config.yml:17]: I guess these get pre-populated somehow https://github.com/marketplace/actions/release-drafter#change-template-variables

Comment (davidangb) [reply .github/release-drafter-config.yml:17]: yup, exactly - they're read from the PR.

**Review by ashanhol — APPROVED** [review]
I don't know github actions that well but the yml looks fine to me

**Pull request merged**